src/integTest/kotlin/io/snyk/SnykBulkFileListenerTest.kt

```diff
@@ -47,13 +55,7 @@ class SnykBulkFileListenerTest : BasePlatformTestCase() {
         super.tearDown()
     }
 
-    private val isContainerEnabledRegistryValue = Registry.get("snyk.preview.container.enabled")
-    private val isContainerEnabledDefaultValue: Boolean by lazy { isContainerEnabledRegistryValue.asBoolean() }
-
-    private fun setUpContainerTest() {
-        imageCache.clear()
-        isContainerEnabledRegistryValue.setValue(true)
-    }
+    /********************* OSS **********************/
```

Comment:
This is probably outside the scope of this PR. But I was thinking that maybe these tests don't belong to `SnykBulkFileListener` but to the caches. So instead of having all things a file change could trigger in this test, it seems more appropriate to me to have test classes for the different 'caches' or 'scan results'. WDYT?

Reply:
Agree. Even more, we need to separate caches form ToolWindowPanel and then tests for that caches will be in separate classes as well.
But for now looks like SnykBulkFileListener is the most cache-responsible class here.